students/saleta_sebastian/lesson_10_organizing_files/filling_in_the_gaps.py

```diff
@@ -0,0 +1,48 @@
+import os
+import re
+
+
+TEST_FILES_DIRECTORY = 'fill_gaps_files'
+FILE_NAME = 'spam'
+TXT_EXTENSION = 'txt'
+
+
+def create_test_data():
+    if not os.path.exists(TEST_FILES_DIRECTORY):
+        os.makedirs(TEST_FILES_DIRECTORY)
+
+    # Create files with gaps
+    for i in range(2, 9):
```

Comment:
I have a bit modified your test data:
```for i in random.sample(range(2, 20), 5):``` and as I can see results are wrong 😄 Could you have look at this one more time?